test/test_broker/acceptance/service-fabrik-api.instances.director.spec.js

```diff
@@ -1862,6 +1862,30 @@ describe('service-fabrik-api', function () {
               mocks.verify();
             });
         });
+        it('should return 201 OK - when rate limiting against bosh is explicitly not required', function () {
```

Comment:
Can you put a similar testcase for 2.0 as well in the 2.0 acceptance test file?

Reply:
Done...